TestAssets/TestProjects/PortableTests/PortableAppWithIntentionalManagedDowngrade/project.json

```diff
@@ -10,9 +10,9 @@
           "type": "platform",
           "version": "1.0.0-rc3-*"
         },
-        "System.Linq": "4.0.0"
+        "System.Linq": "4.1.0-rc3-24125-00"
```

Comment:
@pranavkm - I'm going to have to make a quick change here.  This dependency shouldn't be updated.